package.json

```
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "mocha ./test/* --timeout 10000 --exit",
-    "start": "nodemon server.js"
+    "start": "node server.js",
```

```suggestion
    "start": "node server.js"
```
Woops, forgot to remove the comma